### PR TITLE
fixed small bug with class Item

### PR DESCRIPTION
**Pull request by Osuka42g**

### CODE REVIEW

File: Sources/EliminationMenu.swift [hunk 1]
```diff
@@ -98,23 +98,23 @@ public class EliminationMenu: UIView {
         
         if menu.isLeftAligned {
             view.addConstraint(NSLayoutConstraint(
-                item: menu, attribute: .Left, relatedBy: .Equal, toItem: view, attribute: .Left, multiplier: 1, constant: margin.x)
+                item: view, attribute: .LeftMargin, relatedBy: .Equal, toItem: menu, attribute: .Left, multiplier: 1, constant: margin.x)
```
Comment (r-dent) [inline Sources/EliminationMenu.swift:101]: Changing of `view` and `menu` would reintroduce the bug i fixed in [this Commit](https://github.com/r-dent/EliminationMenu/commit/11973b8bbd3dbda6f76d5c2268c7f3bcaac6d9bb), i think. (the menu was not inset but "outset" of the containing view.)

File: Sources/EliminationMenu.swift [hunk 2]
```diff
@@ -98,23 +98,23 @@ public class EliminationMenu: UIView {
...
             )
         }
         else {
             view.addConstraint(NSLayoutConstraint(
-                item: view, attribute: .Right, relatedBy: .Equal, toItem: menu, attribute: .Right, multiplier: 1, constant: margin.x)
+                item: view, attribute: .RightMargin, relatedBy: .Equal, toItem: menu, attribute: .Right, multiplier: 1, constant: margin.x)
```
Comment (r-dent) [inline Sources/EliminationMenu.swift:106]: I pinned the edges to the view on purpose to [provide compatibility to iOS 7](https://github.com/r-dent/EliminationMenu/commit/16f6f638e8e39c3f8080ac80416ee9b82131405c). Thought that would be a good Idea :-)

File: Sources/EliminationMenu.swift [hunk 3]
```diff
@@ -371,15 +371,15 @@ public class EliminationMenu: UIView {
...
+        public var title: String? = ""
         /// The icon that will be shown in the menu.
         public var icon: UIImage?
         /// The object that you will receive in the selectionHandler.
         public var value: AnyObject!
         /// These insets will also set titleInsets and contentInsets of the items entry button.
         public var iconInsets: UIEdgeInsets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
         
-        init(value:AnyObject, title: String = "", icon: UIImage? = nil) {
+        public init(value value:AnyObject, title: String = "", icon: UIImage? = nil) {
```
Comment (r-dent) [inline Sources/EliminationMenu.swift:382]: I think the fix is the missing `public` right?

**Review by r-dent — CHANGES_REQUESTED** [review]
If the fix is just the missing `public` declaration, maybe you remove the other changes. I would merge it then.

**Comment by Osuka42g** [comment]
Thank you for the feedback r-dent!
I'm making some other modifications, including swift 3 support. I expect to make another pull request soon, also taking in consideration that you told me.

**Comment by r-dent** [comment]
I´m closing this as i think it´s deprecated yet.

**Pull request closed without merging**